### PR TITLE
fixes to reference.md

### PR DESCRIPTION
**Pull request by eileenruberto**
Fixed typos in exact_perfect_match.

<--------------------- REMOVE THE LINES BELOW BEFORE MERGING --------------------->

## Checklist
Please review and complete the following checklist before submitting your pull request:

- [ ] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [ ] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [ ] I have updated the documentation, if applicable.
- [ ] I have added or updated test cases to cover my changes, if applicable.
- [ ] I have minimized the number of reviewers to include only those essential for the review.

## Checklist for Reviewers
Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: docs/site/reference.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ determining it the incoming payload is a match or not to an existing Patient rec
 
 `func:recordlinker.linking.matchers.exact_percent_match`
```
Comment (ericbuckley) [inline docs/site/reference.md:114]:
```suggestion
`func:recordlinker.linking.matchers.exact_perfect_match`
```

**Review by cbrinson-rise8 — APPROVED** [review]
